src/components/button-toggle-group/button-toggle-group.component.js

```diff
@@ -4,13 +4,23 @@ import FormField from '../../__experimental__/components/form-field';
 import ButtonToggleGroupStyle from './button-toggle-group.style';
 import withValidations from '../validations/with-validation.hoc';
 
-const ButtonToggleGroup = withValidations(props => (
-  <FormField { ...props }>
-    <ButtonToggleGroupStyle { ...props }>
-      {props.children}
-    </ButtonToggleGroupStyle>
-  </FormField>
-));
+const ButtonToggleGroup = withValidations((props) => {
+  const { inputWidth, errorMessage, onChange } = props;
+
+  return (
+    <FormField { ...props }>
+      <ButtonToggleGroupStyle
+        aria-label={ props.label }
```

Comment:
I see above that there is an object destructuring . Why wouldn't we add there label and children also?